modules/microshift-ops-config-embed-ostree.adoc

```diff
@@ -0,0 +1,162 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-apply-ops-ostree-offline-use_{context}"]
+= Applying catalogs and Operators in an offline RHEL for Edge image
+
+After you have created a {op-system-ostree} image for offline use and configured {microshift-short} networking settings for offline use, you can configure the namespace and edit the custom resources (CR) generated by the oc mirror dry run for running your Operators.
+
+.Prerequisites
+* You have a {op-system-ostree} image.
+* Networking is configured for offline use.
+* You completed the oc mirror dry run procedure.
+
+.Procedure
+
+. Create a `CatalogSource` custom resource (CR) by using the following example YAML:
++
+[source,terminal]
+----
+$ cat > <my-catalog-source-cr.yaml> <1>
```

Comment:
How is user expected to use this command?

Reply:
Create the actual yaml on the command line.

Reply:
Yes, unless you think better to have users copy our example file into a text file? Just want to provide guidance on the best way to create the file.